app/views/devise/registrations/new.html.erb

```diff
@@ -0,0 +1,34 @@
+<h2>Sign up</h2>
+
+<%= form_for(resource, as: resource_name, url: registration_path(resource_name)) do |f| %>
+  <%= render "devise/shared/error_messages", resource: resource %>
+
+  <div class="field">
+    <%= f.label :email %><br />
+    <%= f.email_field :email, autofocus: true, autocomplete: "email" %>
```

Comment:
- [ OPTIONAL ] Great job on the views but I think it would be great if the user could be able to see button for `signing up` and `signing in` in the home page  so please add them since I just could see the name of the project and a button for create a new post 🔥😄 I could see that you added them after siging up a new user but they should be in the home page 💪🏻